CMakeLists.txt

```diff
@@ -563,6 +579,7 @@ message("")
 message("  Additionally installed third party libraries:")
 invertBoolean(openPMD_USE_INTERNAL_VARIANT openPMD_HAVE_EXTERNAL_VARIANT)
 message("    MPark.Variant: ${openPMD_HAVE_EXTERNAL_VARIANT}")
+message("           Catch2: OFF")
```

Comment:
Should this do the same inversion as the lines above with ```openPMD_USE_INTERNAL_CATCH```?

Reply:
Good question! No, Catch2 will not be installed on `make install` simply because it is only needed at build-time.

Reply:
It took me a while to get the intent of the messages.
That is a really elegant and clever solution and makes the Variant output really useful.

Reply:
Haha, thanks.